server/src/main/java/io/crate/execution/engine/fetch/NodeFetchOperation.java

```diff
@@ -173,7 +172,7 @@ private CompletableFuture<? extends IntObjectMap<StreamBucket>> doFetch(FetchTas
 
         // RamAccounting is per doFetch call instead of per FetchTask/fetchPhase
         // To be able to free up the memory count when the operation is complete
-        final var ramAccounting = ConcurrentRamAccounting.forCircuitBreaker("fetch-" + fetchTask.id(), circuitBreaker);
+        final var ramAccounting = ConcurrentRamAccounting.forCircuitBreaker("fetch-" + fetchTask.id(), circuitBreaker, 0);
```

Comment:
why is it `0` for fetch?

Reply:
We currently don't carry the session settings with the fetch request. But I could add it to the `FetchTask` and then look it up.